SalesforceSDK/Analytics/Store/EventStoreManager.cs

```diff
@@ -162,14 +159,6 @@ public EventStoreManager(string fileNameSuffix, string encryptionKey)
             }
         }
 
-        public async Task ChangeEncryptionKeyAsync(string oldKey, string newKey)
```

Comment:
As we discussed offline, the `Encryptor` needs to trigger changing the encryption key used to encrypt the files here. So we'll need to bring back the re-encryption piece here, and add those updates to `Encryptor` to trigger it here.